docs/en/build.md

```diff
@@ -137,8 +137,11 @@ Each package's installation command is given based on Ubuntu 18.04.
 
   A high-performance image processing library of openPPL supporting x86 and cuda platforms.</br>
   It is **OPTIONAL** which only be needed if `cuda` platform is required.
-  ```bash
-  git clone git@github.com:openppl-public/ppl.cv.git
+  
+  Using v0.6.1, since latest updates have broughtup breaking changes
+  ```Bash
+  wget https://github.com/openppl-public/ppl.cv/archive/refs/tags/v0.6.1.zip
```

Comment:
```suggestion

  Using v0.6.1, since latest updates have broughtup breaking changes
  ```Bash
  wget https://github.com/openppl-public/ppl.cv/archive/refs/tags/v0.6.1.zip
```